.codespellrc

```diff
@@ -0,0 +1,7 @@
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+[codespell]
+skip = .git,*.svg,*.css,*.min.*,*-min.js,.codespellrc,locale,static,kickstarter-announcement.md
+check-hidden = true
```

Comment:
`check-hidden` is not a good idea on a GitHub repo because it will spellcheck all our dependencies and a bunch of other files that we don't have the desire or ability to modify.  If we remove that option then we can simplify to:
`codespell --ignore-words-list=fo,malcom,ser --skip="*/kickstarter-announcement.md,*.js,*.map,*.po"`

Also, this project already has a `setup.cfg` file so perhaps it would be better to put this configuration there.
This has the added advantage that it will be automatically migrated to `pyproject.toml` by tools like `ini2toml`.

Reply:
- moved configuration to `setup.cfg` (yet to adjust my https://github.com/yarikoptic/improveit/blob/main/codespellit to do so automagically -- it only considers pyproject.toml ATM)
- minimized to your recommendation/liking
- dropped added workflow entirely -- with pre-commit workflow you don't need it separately (checked that pre-commit workflow did pickup newly added typo)